src/include/stir/IO/GEHDF5Wrapper.h

```diff
@@ -53,44 +55,58 @@ class GEHDF5Wrapper
 {
 public:
 
+    //! check signature to see if this is a GE RDF9 or higher file
+    /*! calls current_GE_signature(H5::H5file&) */
     static bool check_GE_signature(const std::string& filename);
 
-    static bool check_current_signature(H5::H5File& file);
+    //! check signature to see if this is a GE RDF9 or higher HDF5 file
+    /*! does minimal checks to see if expected datasets are present in the HDF5 object */
+    static bool check_GE_signature(H5::H5File& file);
```

Comment:
You can not overload static methods, right?

Reply:
sure. why not? You cannot use `virtual` for `static` functions (so you cannot "override" them in a derived class).

Reply:
Ah, google failed me and gave me a StackOverflow question about static vs non-static. My bad.